sources/Renderer/OpenGL/Platform/MacOS/MacOSGLContext.mm

```diff
@@ -68,6 +70,15 @@
 
 void MacOSGLContext::Resize(const Extent2D& resolution)
 {
+    GLint dim[2];
```

Comment:
Looks good to me, but again, I'm not experienced enough with the CGL functions to provide a sophisticated feedback on this one. But I'd appreciate a bit explanation.

Reply:
I'm having difficulty finding documentation on this function. The only write up is at the following URL and it basically boils down to "call these routines to change the back buffer sizes for OpenGL". https://developer.apple.com/library/archive/documentation/GraphicsImaging/Conceptual/OpenGL-MacProgGuide/opengl_contexts/opengl_contexts.html#//apple_ref/doc/uid/TP40001987-CH216-SW28